planning/behavior_path_planner/include/behavior_path_planner/scene_module/side_shift/side_shift_module.hpp

```diff
@@ -46,6 +50,27 @@ struct SideShiftParameters
   double drivable_area_height;
 };
 
+struct SideShiftRequestTimer
+{
+  time_point<high_resolution_clock> last_request_clock = high_resolution_clock::now();
+
+  bool override_request = true;
+
+  bool isRequestAllowed(const double & change_time)
+  {
+    time_point<high_resolution_clock> current_request_clock = high_resolution_clock::now();
+    const auto requested =
+      duration<double, std::milli>(current_request_clock - last_request_clock).count() * 0.001;
```

Comment:
If the name is requested, I think it's bool.
interval_from_last_time_msec is better, maybe

Reply:
Thank you. yes, `interval_from_last_time_msec` is much better.